e2e/core/settings-component.e2e.ts

```diff
@@ -0,0 +1,119 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { LoginPage } from '../pages/adf/loginPage';
+import { SettingsPage } from '../pages/adf/settingsPage';
+import { browser, element, protractor } from 'protractor';
+import TestConfig = require('../test.config');
+import { AcsUserModel } from '../models/ACS/acsUserModel';
+
+describe('Settings component', () => {
+
+    const loginPage = new LoginPage();
+    const settingsPage = new SettingsPage();
+
+    let adminUserModel = new AcsUserModel({
+        'id': TestConfig.adf.adminUser,
+        'password': TestConfig.adf.adminPassword
+    });
+
+    describe('Settings component', () => {
+        beforeEach( (done) => {
+            settingsPage.goToSettingsPage();
```

Comment:
Can you add a check to make sure that the settings page is displayed ?

Reply:
inside the method itself, it's checking that the dropdown on the page is displayed -> Util.waitUntilElementIsVisible(this.providerDropdown);